index.js

```diff
@@ -106,6 +107,10 @@
       } else if (  ( X.lookup('foam.ui.View').isInstance(obj) )
                    || ( 'CView' in GLOBAL && CView.isInstance(obj) ) ) {
         view = obj;
+      } else if ( X.lookup('foam.u2.Element').isInstance(obj) ) {
+        document.body.insertAdjacentHTML('beforeend', obj.outerHTML);
+        obj.load();
+        return;
       } else if ( obj.toView_ ) {
```

Comment:
I've added a TODO for now. There doesn't seem to be anything that has a `toE` but isn't an `Element` itself.